rust/libceed/src/lib.rs

```diff
@@ -707,6 +707,71 @@ impl Ceed {
         )
     }
 
+    /// Returns an $H(div)$ Basis
+    ///
+    /// # arguments
+    ///
+    /// * `topo`    - Topology of element, e.g. hypercube, simplex, ect
+    /// * `ncomp`   - Number of field components (1 for scalar fields)
+    /// * `nnodes`  - Total number of nodes
+    /// * `nqpts`   - Total number of quadrature points
+    /// * `interp`  - Row-major `(nqpts * dim * nnodes)` matrix expressing the
+    ///                 values of basis functions at quadrature points
+    /// * `div`     - Row-major `(nqpts * nnodes)` matrix expressing the
+    ///                 divergence of basis functions at quadrature points
+    /// * `qref`    - Array of length `nqpts` holding the locations of quadrature
+    ///                 points on the reference element `[-1, 1]`
+    /// * `qweight` - Array of length `nqpts` holding the quadrature weights on
+    ///                 the reference element
```

Comment:
Can we add tests too to verify that the bindings function as intended?

Reply:
I imagine you are also looking for Python tests and I should just duplicate the existing simplex H(div) and H(curl) C tests there?